src/components/WelcomeView.jsx

```diff
@@ -82,11 +83,16 @@ class WelcomeView extends React.PureComponent {
           Welcome to the Jade Data Repository
           </div>
           <div>
-            <GoogleLogin // stealing clientId from Terra
+            <GoogleLogin // TODO this component may be unuseable once we require a terra registration
```

Comment:
This will still be required -- the user needs to have a google account AND be a terra user. We'll need to do the terra check separately after we confirm they are logged in.